src/actions/background/IBackgroundAction.ts

```diff
@@ -0,0 +1,5 @@
+import {IAction} from "../../interfaces/IAction";
+
+export interface IBackgroundAction extends IAction{
```

Comment:
I know we used these interfaces before, but I'd like to question if we really need them.
Can this be removed?